circle.yml

```diff
@@ -1,3 +1,16 @@
 machine:
   node:
-    version: 6.5.0
+    version: 6.10.1
+dependencies:
```

Comment:
From the circle docs: I think it supports it officially in that we don't have to worry about yarn being globally installed. https://circleci.com/docs/1.0/yarn/